operator/version/version.go

```diff
@@ -41,7 +41,10 @@ func init() {
 	// If dockerinfo has a tag (e.g., specified by LDFlags), we will use it as the version of operator
 	tag := buildversion.DockerInfo.Tag
 	if pkgversion.IsVersionString(tag) {
-		OperatorVersionString = tag
+		OperatorVersionString, err = pkgversion.TagToVersionString(tag)
+		if err != nil {
+			panic(err)
```

Comment:
why panic instead of proper handling?

Reply:
ideally we should return error, but this is an init(). I am not quite sure what to do, so just followed the same pattern in other places of this function.

Reply:
Seems like this is actualy impossible to trigger. TagToVersionString can only error if its not a semver and we check it is in IsVersionString before hand. Maybe we should just make a function that is safe and never returns an error - instead returning the original tag if its not semver? seeing `panic` is a red flag

Reply:
agree.